cypress/integration/homework/calculator.spec.js

```diff
@@ -0,0 +1,158 @@
+// <reference types="Cypress" />
+
+import HomePage from './classes/HomePage.js'
+
+var homePage = new HomePage
+
+var website = "https://testsheepnz.github.io/BasicCalculator";
+
+var numberField1 = "#number1Field";
+var numberField2 = "#number2Field";
+var clearButton = "#clearButton";
+var buildDropDown = "#selectBuild";
+var actionDropDown = "#selectOperationDropdown";
+var action = "";
+
+describe('Calculator Homepage', () => {
+    describe('Addition', () => {
+        it.only("Test addition on every version of a calculator", () => {
+            action = "Add";
+            cy.visit(website)
+            cy.get(actionDropDown).select(action)
+            for (var i = 1; i <= 9; i++) {
```

Comment:
In this specific task, it is not a good way to use a loop inside one test to go through all builds. Why? Because if any of the builds failed (e.g. in this case 2nd build failed), other builds are not checked at all. 
How this can be fixed? Just move the loop above the test initiation (e.g. above **it**). This means that you will have 9 same tests with 9 builds (1 build in 1 test) instead of having 1 test with 9 builds inside.